test/clj_kondo/unused_import_test.clj

```diff
@@ -81,4 +82,19 @@
     (is (empty? (lint! "(import '[java.util Foo]) (defn foo [^Foo x] x)")))
     (is (empty? (lint! "(import '[java.util Foo]) (try 1 (catch Foo _e nil))")))
     (is (empty? (lint! "(ns foo (:import [foo Bar])) (defn foo [x] x) (defn bar [x] (-> x ^Bar (.execute)))"))))
-)
+  (testing "return import info"
+    (assert-submaps
+      '({:type :unused-import,
+         :message "Unused import Date",
+         :level :warning,
+         :row 1,
+         :end-row 1,
+         :end-col 33,
+         :col 29,
+         :import java.util.Date
```

Comment:
I suggest renaming `:import` to `:class` and leave out the `:class` since this is redundant info.